app/lib/methods/getRoles.js

```diff
@@ -20,41 +21,38 @@ export async function onRolesChanged(ddpMessage) {
 		const db = database.active;
 		const rolesCollection = db.get('roles');
 		try {
-			const rolesRecord = await rolesCollection.find(_id);
-			try {
```

Comment:
If something bad happens inside the block, we would try to re-create the role again, which would raise a SQLite PK error.